config/base/scene/simple_desktop.json

```diff
@@ -241,6 +235,79 @@
           "texture": "../share/resources/textures/neptune.jpg"
         }
       }
+    },
+	"csp-simple-wms-bodies": {
+      "bodies": {
+        "Earth": {
+          "texture":  "../share/resources/textures/earth.jpg",
+          "wms": [
+            {
+              "name": "Fires",
```

Comment:
Maybe extend the name with sensor sources (MODIS?, VIIRES? sensor) .e.g. Fire-Hotspots-MODIS

Reply:
Where do I find this sensor sources?

Reply:
https://firms.modaps.eosdis.nasa.gov/